### PR TITLE
robot PRs pass validation

### PR DESCRIPTION
**Pull request by reaganthomas**
Ignoring branch and commit checks if there is a single commit by a robot.

### CODE REVIEW

File: action.js [hunk 1]
```diff
@@ -82,15 +107,17 @@ module.exports = class {
   async validate(type, validIssueTypes) {
     let valid = false;
     switch (type) {
-      case 'all':
-        valid = (await this.validateCommitsHaveIssueIds()) && this.validateBranchHasIssueId();
+      case "all":
```
[the remark this answers is not in the document]
Comment (reaganthomas) [reply action.js]: I'm not sure what caused that.....but I have your plugins installed :D

File: action.js [hunk 2]
```diff
@@ -38,36 +39,60 @@ module.exports = class {
...
+    );
+  }
+
   async validateCommitsHaveIssueIds() {
+    let valid = true;
     const commits = await this.getCommits();
+
+    if (this.containsRobotCommits(commits)) {
+      return valid;
+    }
```
Comment (reaganthomas) [inline action.js]: This was a main change

File: action.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (reaganthomas) [reply action.js]: Looks like I'm just using w/e is on the repo, but I could be mistaken. Not sure how to debug this.

**Comment by duanebester** [comment]
So you'll always want to run `npm run build` before adding and committing final code

**Pull request merged**